### PR TITLE
Docs: remove brace expansion from configuring.md

### PR DESCRIPTION
**Pull request by BarryThePenguin**
Remove mention of brace expansion after conversation in #5314

I added a section about the specific issue I was having. If you think it's not necessary I can remove it :+1:

### CODE REVIEW

File: docs/user-guide/configuring.md [hunk 1]
```diff
@@ -652,14 +651,23 @@ For example, placing the following `.eslintignore` file in the current working d
...
+### Ignore Patterns
+
+* Brace expansion is not supported in the `.gitignore` specification. An alternative is to match multiple files with multiple patterns. For example, the equivelant of `**/*.{js,ts,coffee}` would be:
+
+```
+# Ignore multiple patterns
+eslint . --ignore-pattern **/*.js --ignore-pattern **/*.ts --ignore-pattern **/*.coffee
+
+# Include multiple extensions
+eslint . --ext js --ext ts --ext coffee
```
Comment (IanVS) [inline docs/user-guide/configuring.md]: It seems this line is not related to ignore patterns, not sure it belongs here.

Comment (BarryThePenguin) [inline docs/user-guide/configuring.md]: No problem, happy to remove it. This was what I was having problems with, so I thought I'd mention it

**Comment by IanVS** [comment]
Thanks for the PR!  I'm not sure the added section is necessary here, but what I would love to see is a section added to the "Command Line Interface" page for `--ignore-pattern`.  I think that's where this section would belong.

**Comment by BarryThePenguin** [comment]
Great! I've removed that section. Will create a new PR once I've had a read through the Command Line Interface page

**Comment by nzakas** [comment]
Please add "(refs #5314)" to the end of your commit summary

**Comment by BarryThePenguin** [comment]
Done :+1:

**Comment by IanVS** [comment]
I hate to say this, but you added `(refs #5314)` to the end of your commit body, not the summary (first line).  The reason this matters is the tool we use to generate our changelog will pick up on issue references in the first line (summary), but not the rest of the body.  If you wouldn't mind amending the commit message one more time to move that to the first line, I think this will be G2G.  Thanks for sticking with it.  :smiley:

**Comment by BarryThePenguin** [comment]
Sure, I misread the guidelines. I've updated and rebased :+1:

**Comment by nzakas** [comment]
LGTM. Thanks.
